index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link type="text/css" rel="stylesheet" href="./index.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Landing Page</title>
+</head>
+<body>
+    <section class="navbar">
+        
+        <nav class="one">
+                <a href="#" class="normal">Home</a>
+                <a href="#" class="normal">About</a>
+                <a href="#" class="normal">Products</a>
+                <a href="#" class="normal">Contact</a>
+                <a href="#"><button class="log">Log in</button></a>
+                <a href="#"><button class="sign">Sign up free</button></a>
+        </nav>
+    </section>
+
+    <footer>
+        <div>
+            <div class="flex4">
+                <div class="logosm"><img class="small-logo" src="https://eep.io/images/yzco4xsimv0y/5daYAeMRSgIUUiSquGwsgI/1904e3f818575ec9625fa15dab193243/icon-facebook.svg?w=320&fm=webp&q=80" alt=""></div>
+                <div class="logosm"><img class="small-logo" src="https://eep.io/images/yzco4xsimv0y/3VjJhbqftKuok6gqYuUQKy/328d56c5d13ad50b9679b11e716bb218/icon-twitter.svg?w=320&fm=webp&q=80" alt=""></div>
+                <div class="logosm"><img class="small-logo" src="https://eep.io/images/yzco4xsimv0y/RlNw7Xkw8uKYAUoYU4aQw/ff3eada8736da9e63fbd9f929c6a677c/icon-instagram.svg?w=320&fm=webp&q=80" alt=""></div>
+                <div class="logosm"><img class="small-logo" src="https://eep.io/images/yzco4xsimv0y/n5saFR4miW4U4ygUIQo0A/8f9c60cb133b80fda2fe9bdb825b4a5f/icon-linkedin.svg?w=320&fm=webp&q=80" alt=""></div>
+                <div class="logosm"><img class="small-logo" src="https://eep.io/images/yzco4xsimv0y/2z1ra9BrgQewIqMQckMasg/58e3e8ec04acd60f0642f89dfbd71218/icon-youtube.svg?w=320&fm=webp&q=80" alt=""></div>
+                <div class="logosm"><img class="small-logo" src="https://eep.io/images/yzco4xsimv0y/7nawBEArBKpxRukK66OZtN/af9879112b20982e13748d47b24c8681/pinterest-logo.svg?w=320&fm=webp&q=80" alt=""></div>
+            </div>
+        </div>
+    </footer>
```

Comment:
- [ ]  I would implore you to use ul and li for categories in your tag to engulf your anchor tag instead of div tags.

Reply:
- [ ] Good job @Amazinggracee 😄, but you forgot to highlight the effort of the student, you could say something like 
    
   - Good job for using `div` tag 👍 , but  I would implore you to use ul and li for categories in your tag to engulf your anchor tag instead of div tags.